src/bundle/Controller/LanguageController.php

```diff
@@ -107,8 +107,9 @@ public function listAction(Request $request): Response
         return $this->render('@EzPlatformAdminUi/admin/language/list.html.twig', [
             'pager' => $pagerfanta,
             'form_languages_delete' => $deleteLanguagesForm->createView(),
-            'canEdit' => $this->isGranted(new Attribute('content', 'translations')),
-            'canAssign' => $this->isGranted(new Attribute('content', 'translations')),
```

Comment:
Isn't this a BC break?

Reply:
Yes. We should not freely rename Twig variables because it will break custom templates. I opt for adding both variables + camelCase can take a deprecation comment. What do you think?

Reply:
@mikadamczyk but please add this deprecation comment, so we can remove this variable in 2.3 release 😸